src/web/services/student.service.ts

```diff
@@ -15,12 +16,33 @@ export class StudentService {
   }
 
   /**
-   * Gets all students in a course as an instructor by calling API.
+   * Get a list of students of a course by calling API.
+   * If teamName is provided, only students in that team will be returned.
+   * Otherwise, all students in the course will be returned.
    */
-  getStudentsFromCourse(courseId: string): Observable<Students> {
+  getStudentsFromCourse(courseId: string, teamName?: string): Observable<Students> {
+    if (teamName) {
+      const paramsMap: { [key: string]: string } = {
+        courseid: courseId,
+        teamname: teamName,
+      };
+      return this.httpRequestService.get('/students', paramsMap);
+    }  {
```

Comment:
```suggestion
    } else {
```
?

Reply:
I tried to add "else" but then `npm lint` removes it 😅